script/Ciel/Day 13/13_01_01_CIEL13_1B.txt

```diff
@@ -26,7 +26,7 @@ The woman gave up and left.
 [sha:ab724912461890493cb1942da69474608886e048]{
 -- Page 10020, Offset 33229.
 -- 代わる代わる違う女がやってきて扉をノックしたが、すべて無視した。
-The women took turns in coming and knocking on the door, but I ignored all of it.
+The women took turns coming and knocking on the door, but I ignored all of them.
```

Comment:
Each woman in the house came in turn and knocked on my door, but I ignored them all.